staging/src/k8s.io/kubectl/pkg/cmd/create/create_secret_docker.go

```diff
@@ -58,6 +59,9 @@ var (
 		  kubectl create secret docker-registry my-secret --docker-server=DOCKER_REGISTRY_SERVER --docker-username=DOCKER_USER --docker-password=DOCKER_PASSWORD --docker-email=DOCKER_EMAIL
 
 		  # Create a new secret named my-secret from ~/.docker/config.json
+		  kubectl create secret docker-registry my-secret --from-file=path/to/.docker/config.json
+
+		  # Create a new secret named my-secret from ~/.docker/config.json with a key name
```

Comment:
Do we even want to keep this example?

Even though we support this way, it isn't necessary to specify `.dockerconfigjson` anymore, right?

It seems that we should be guiding people to use the simpler syntax.

Thoughts anyone?